src/Avatar.tsx

```diff
@@ -12,33 +11,31 @@ type StyledAvatarProps = {
   src: string
   /** Provide alt text when the Avatar is used without the user's name next to it. */
   alt?: string
-} & SxProp
+} & SxProp &
+  Omit<React.ComponentProps<'img'>, 'ref'>
```

Comment:
We could use `ComponentPropsWithoutRef` here

```suggestion
  React.ComponentPropsWithoutRef<'img'>
```